packages/thirdweb/src/exports/react.ts

```diff
@@ -199,3 +199,6 @@ export type {
 // Site Embed and Linking
 export { SiteEmbed } from "../react/web/ui/SiteEmbed.js";
 export { SiteLink } from "../react/web/ui/SiteLink.js";
+
+// Account
+export { Account } from "../react/web/ui/prebuilt/Account/index.js";
```

Comment:
This causes problems in the docs because we already have a type called Account.

Check out what I did in my docs pr for the NFT component. Basically I use a namespaced export instead with a barrel file, and call the main provider NFT.Provider

Reply:
yesser